src/library/Uno.Material/Converters/ToUpperConverter.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Windows.UI.Xaml.Data;
+
+namespace Uno.Material.Converters
+{
+	public class ToUpperConverter : IValueConverter
+	{
+		public object Convert(object value, Type targetType, object parameter, string language)
+		{
+			var str = value as string;
+			return string.IsNullOrEmpty(str) ? string.Empty : str.ToUpper();
```

Comment:
With the way it is being used in Button, it will brick the control if anyone decide to anything no-string in it.
Like <TextBlock Text="because they can" />, or complex object + ContentTemplate.

Reply:
good point @Xiaoy312

Reply:
you want this:
```cs
return value is string text ? text.ToUpperInvariant() : value;
```

Reply:
yeah just realized that, should be fixed now